goltsmannn2/default.nix

```diff
@@ -0,0 +1,8 @@
+{ stdenv, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "lab2_part1";
+  version = "0.1.0";
+  src = "CMakeLists.txt";
```

Comment:
this should just be the relative path to your source code, not just a single source:

https://blog.ielliott.io/nix-docs/mkDerivation.html

Reply:
more docs here: https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv/stdenv.chapter.md